docs/reference/api/resources/paths.rst

```diff
@@ -0,0 +1,55 @@
+App Paths
+==========
+
+A mechanism for obtaining platform-appropriate file system locations for an
+application.
+
+.. rst-class:: widget-support
+.. csv-filter:: Availability (:ref:`Key <api-status-key>`)
+   :header-rows: 1
+   :file: ../../data/widgets_by_platform.csv
+   :included_cols: 4,5,6,7,8,9
+   :exclude: {0: '(?!(App Paths|Component)$)'}
+
+Usage
+-----
+
+When Python code executes from the command line, the working directory is a
+known location - the location where the application was started. However, when
+building GUI apps, there is no working directory. As a result, when specifying
+file paths, relative paths cannot be used, as there is no location to which they
+can be considered relative.
+
+Complicating matters further, operating systems have conventions (and in some
+cases, hard restrictions) over where certain file types should be stored. For
+example, macOS provides the ``~/Library/Application Support`` folder; Linux
+encourages use of the ``~/.config`` folder (amongst others), and Windows
+provides the ``AppData/Local`` folder in the user's home directory. Application
+sandbox and security policies will prevent sometimes prevent reading or
+writing files in any location other than these pre-approved locations.
+
+To assist with finding an appropriate location to store application files, every
+Toga application has a ``paths`` object that provides known file system
```

Comment:
This page doesn't explain how to actually get the `paths` object. There should be a link to the `toga.App.paths` documentation page, probably using `:any:` or something similar.